### DISCUSSION TITLE
Richieste

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#537] [Richieste di aiuto]
Salve una cosa il sito non mi è molto chiaro quindi mi potreste dire come installare il registro perfavore
Savide coco

**Comment by trinko** [comment]
Vedi la [documentazione](https://iisgiua.github.io/giuaschool-docs/install.html) oppure chiedi [aiuto per l'installazione](https://github.com/iisgiua/giuaschool/discussions/504)

**Reply by giadaschiavone** [reply]
No, l'aiuto per l'installazione non risponde nessuno e/o dicono che lo fanno alle vere scuole e altre cose, in teoria, non lo fa a nessuno. Sembrerebbe una truffa e/o solo per mettersi in mostra. Comunque, vedi la documentazione per l'installazione. Ricordati di inserire il php 8.2.

**Comment by giadaschiavone** [comment]
No, l'aiuto per l'installazione non risponde nessuno e/o dicono che lo fanno alle vere scuole e altre cose, in teoria, non lo fa a nessuno. Sembrerebbe una truffa e/o solo per mettersi in mostra. Comunque, vedi la documentazione per l'installazione. Ricordati di inserire il php 8.2.

**Comment by fabriziotoppi** [comment]
mi sono  trovato nella stessa tua situazione .ho chiesto aiuto ad un contatto che si mettva a disposizione . mi ha chiseto 100  cose . Devo dire la verita, anche a me è sembrato un contatto strano. Alla fine sono riuscito da solo e ora funziona  anche grazie a trinko che mi ha risposto ad alcuni interrogativi. sono a disposizione se vuoi
